### DISCUSSION TITLE
Migração de ser Developer para Gestor / PO

### DISCUSSION THREAD

**Discussion by ocaioaugusto** [frontendbr/forum#1527] [Discussão]
Eu tenho pensado bastante sobre a carreira de developer no geral. Eu trabalho há quase um ano oficialmente com desenvolvimento e ultimamente algumas coisas e dúvidas têm brotado na minha cabeça.
O que acontece é que eu tô me interessando bastante pela parte mais analítica de projetos, a área de planejamento, inovação, etc. Me sinto muito preso e sufocado sendo um developer, e sinto que o meu teto é muito mais baixo do que as outras partes do projeto. 
Gostaria de saber tanto uma opinião de vocês sobre e, pra quem fez o caminho, como começar a migrar entre esses dois postos. O conhecimento é um pouco distinto das habilidades técnicas que aprendemos para desenvolver e eu to um pouco perdido de por onde começar. 

Valeu!

**Comment by kavalcante** [comment]
Fala mano! Ótima pergunta. 

Estou nesse momento durante essa transição. No meu caso, eu co-fundei uma startup, então esse caminho está sendo um pouco mais natural e menos dolorido. Ainda trabalho uma boa parte da minha sprint como dev (70% atualmente) e estou progressivamente aumentando a parte de gestão.

Meu cenário é bem parecido com o seu, aparentemente. Eu gosto muito de produto, por isso estou me tornando um PM.

A dica é: comece estudando já, para isso existem livros bem legais como o [INSPIRED](https://www.amazon.com.br/Inspired-Create-Tech-Products-Customers/dp/1119387507/ref=asc_df_1119387507/?tag=googleshopp00-20&linkCode=df0&hvadid=379795170134&hvpos=1o1&hvnetw=g&hvrand=13804704626581589718&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=1001726&hvtargid=pla-387522764878&psc=1), The Product Book da Product School (de graça e em ptBR [aqui](https://www.productschool.com/the-product-book/)), [Intercom on Product Management Book](https://www.intercom.com/books/product-management).

Depois que começar a se aprofundar nesse assunto, é procurar uma vaga nessa área (seja na sua empresa, o que muitos fazem) ou em outra empresa.

**Comment by ocaioaugusto** [comment]
> Fala mano! Ótima pergunta.
> 
> Estou nesse momento durante essa transição. No meu caso, eu co-fundei uma startup, então esse caminho está sendo um pouco mais natural e menos dolorido. Ainda trabalho uma boa parte da minha sprint como dev (70% atualmente) e estou progressivamente aumentando a parte de gestão.
> 
> Meu cenário é bem parecido com o seu, aparentemente. Eu gosto muito de produto, por isso estou me tornando um PM.
> 
> A dica é: comece estudando já, para isso existem livros bem legais como o [INSPIRED](https://www.amazon.com.br/Inspired-Create-Tech-Products-Customers/dp/1119387507/ref=asc_df_1119387507/?tag=googleshopp00-20&linkCode=df0&hvadid=379795170134&hvpos=1o1&hvnetw=g&hvrand=13804704626581589718&hvpone=&hvptwo=&hvqmt=&hvdev=c&hvdvcmdl=&hvlocint=&hvlocphy=1001726&hvtargid=pla-387522764878&psc=1), The Product Book da Product School (de graça e em ptBR [aqui](https://www.productschool.com/the-product-book/)), [Intercom on Product Management Book](https://www.intercom.com/books/product-management).
> 
> Depois que começar a se aprofundar nesse assunto, é procurar uma vaga nessa área (seja na sua empresa, o que muitos fazem) ou em outra empresa.

Faço analise de sistemas e to me encantando cada vez mais pela engenharia de software. A parte mais difícil pra mim mesmo está sendo achar conteúdo prático pra estudar esse tipo de coisa fora da faculdade. Tô pensando em fazer aqueles bootcamps de um fim de semana que aparecem de vez em quando por ai, vale a pena?

Valeu pela resposta, cara. Vejo que o caminho mais natural pra esse tipo de coisa seria cofundando alguma empresa mesmo, sendo que eu acho um pouco mais difícil coordenar algo no próprio trabalho não sendo sênior

**Comment by kavalcante** [comment]
@ocaioaugusto Cara, não sei te dizer pq nunca fiz um. E como eu estou empreendendo agora, grana não é algo que tá tendo com frequência hahaha (primeiro a empresa, dps os sócios).

Mas sei que tem escolas bastante renomadas como a Product School, por exemplo. Recomendo tu começar lendo algum desses livros, por exemplo. Ouvir podcasts, como o Product Backstage (PT-BR) ou o This is Product Management pra você entender se esse é o caminho.

Gerir tecnologia possuem muitas possibilidades diferentes e muitos caminhos também. Tenho certeza que algum deles você vai gostar mais que os outros.

**Comment by ocaioaugusto** [comment]
> @ocaioaugusto Cara, não sei te dizer pq nunca fiz um. E como eu estou empreendendo agora, grana não é algo que tá tendo com frequência hahaha (primeiro a empresa, dps os sócios).
> 
> Mas sei que tem escolas bastante renomadas como a Product School, por exemplo. Recomendo tu começar lendo algum desses livros, por exemplo. Ouvir podcasts, como o Product Backstage (PT-BR) ou o This is Product Management pra você entender se esse é o caminho.
> 
> Gerir tecnologia possuem muitas possibilidades diferentes e muitos caminhos também. Tenho certeza que algum deles você vai gostar mais que os outros.

Pô cara, valeu mesmo por compartilhar. Vou me integrar mais nesse mundo a partir do que você me indicou. É meio demorado mas acho que vale muito apena.
E boa sorte ai na sua jornada, mano!

Abraço!
